README.md

```diff
@@ -7,35 +7,129 @@ This library is currently used in the [tgstation] codebase, and is required for
 A pre-compiled DLL version can be found in the repo root, but you can build your own from
 this repo at your preference.
 
+## Dependencies
+
+The [Rust] compiler:
+
+1. Use [the Rust installer](https://rustup.rs/), or another Rust installation method,
+   or run this command:
+
+    ```sh
+    curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain stable
+    ```
+
+1. Set the default compiler to **32-bit**:
+
+    ```sh
+    # in the `rust-g` directory...
+    cd rust-g
+    # Linux
+    rustup override add stable-i686-unknown-linux-gnu
+    # Windows
+    rustup override add stable-i686-pc-windows-msvc
+    ```
+
+OpenSSL (**Optional**, not required by the default configuration):
+
+* Ubuntu and Debian users run:
+
+    ```sh
+    sudo apt-get install libssl-dev:i386 libssl1.0.0:i386 pkg-config gcc-multilib
```

Comment:
libssl-dev:i386 includes libssl1.0.0:i386

Reply:
This won't always be the case.